nanome/_internal/_structure/_helpers/_conformer_helper.py

```diff
@@ -139,6 +138,7 @@ def convert_to_conformers(complex, force_conformer = None): #Data.Complex -> Dat
                     off = 0
                     if name_hash in names_dictionary:
                         off = names_dictionary[name_hash]
+                    off +=1
```

Comment:
This was a huge bug with the to_conformer code that basically caused all atoms in a residue of the same element to become 1 atom in the conformer.